### ISSUE TITLE
Design & Define guided market research

### ISSUE THREAD

**Issue by janelx** [18F/gsa-small-business-experience#71]
As a potential vendor:
1. I would like to know if working with GSA is worth it for me?

- **a. Getting on schedule" may be a good fit for me but how do I know I will be successful as a vendor?** This issue
- b. Subcontracting next steps? (link to new issue)

- a. Design solution: guided market analysis

- Is there need for my goods and/or services?
- Will I be & what do I need to be competitive against current and potential vendors?

- [x] Research: what is the needed to for competitive market analysis? 
   * [Feel like these notes cover this](https://docs.google.com/document/d/1Uz4o6XCrUTTgF1u-qbJvq-kwl0q5ovbpX7axAS44GPs/edit#heading=h.73kwi7qevecp)
- [x] Design Guided Market analysis
  * [Prototype for testing in this google doc](https://docs.google.com/document/d/1wRMA5Bq92EZHlNiQwMZPLkGyCnBLlOBSCXjyYWMXJXw/edit#heading=h.y3qu01wuhqc9)

Done when
- [x] Design review between @cmajel and @janelx 
- [x] Review with @jduss4 and update "for Jess" figma board with assets and interactions

**Comment by cmajel** [comment]
Ok! I did some iterations in Figma that were reviewed with @janelx , and [have moved an updated version to this Google Doc](https://docs.google.com/document/d/1wRMA5Bq92EZHlNiQwMZPLkGyCnBLlOBSCXjyYWMXJXw/edit#heading=h.y3qu01wuhqc9).

A couple of things:
* For testing, @janelx and I discussed how we'd like to link to a PDF of this document in the prototype. This is not a high testing priority, but we'd like to get feedback if the test leads them there. @jduss4 is this possible?
* I added a couple of notes of things I do not know or that need revisiting. Comments and direct edits welcome; I also think these things could come later. The biggest ones are:
  * Guidance on how to find your PSC code
  * Guidance on using the forecasting tool for upcoming contracts
